### PR TITLE
Zth setup

### PR DESCRIPTION
**Pull request by zachholcomb**
Incorporated team and game class into stat_tracker class so that stat_tracker creates array of team and game instances.

### CODE REVIEW

File: data/teams.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
-team_id,franchiseId,teamName,abbreviation,Stadium,link
```
Comment (zachholcomb) [inline data/teams.csv:1]: had to change headers in teams.csv to follow better ruby syntax

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -16,12 +19,14 @@ def self.from_csv(locations_params)
 
     games_array = []
     CSV.foreach(games_file, headers: true, header_converters: :symbol) do |row|
-      games_array << row
+      game = Game.new(row)
+      games_array << game
```
Comment (zachholcomb) [inline lib/stat_tracker.rb:23]: now with the game and team class we can create arrays of game and team instances with better searching capabilities

File: lib/stat_tracker.rb [hunk 3]
```diff
@@ -16,12 +19,14 @@ def self.from_csv(locations_params)
...
     end
 
     teams_array = []
     CSV.foreach(teams_file, headers: true, header_converters: :symbol) do |row|
-      teams_array << row
+      team = Team.new(row)
+      teams_array << team
```
Comment (zachholcomb) [inline lib/stat_tracker.rb:29]: same here, incorporated team class

File: data/teams.csv (hunk 1 above)
Comment (elguapogordo) [reply data/teams.csv:1]: horrible syntax. NOT ON MY WATCH, aurora, megan, and tim!

**Pull request merged**